### PR TITLE
UserProfileEditPage

### PR DESCRIPTION
**Pull request by Banychen**
Only Add UserProfileEditPage. Still figuring out how to upload image.

### CODE REVIEW

**Comment by bob-zs** [comment]
@Banychen Maybe you can try profs demo? 
https://youtu.be/Jh0zP9dkUsE

**Comment by Banychen** [comment]
> 

Thank you ! I was actually finding the recording of that day.

**Comment by bob-zs** [comment]
Np, always feel free to ask or let us know when you need to find something.

File: src/store/auth.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+  switch (action.type) {
+    case 'SET_AUTH':
+    return state=action.auth;
+    case 'UPDATE_AUTH':
+    return {...state, auth:action.auth};
+    default:
+    return state
+  }
+};
+*/
```
Comment (bob-zs) [inline src/store/auth.js]: Delete Please

**Review by Diane-Sayos — APPROVED** [review]

**Pull request merged**